### PR TITLE
Coil:Cooling:DX cleanup

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
Coil:Cooling:DX cleanup to address comments from the v9.5 dx coil update #8558.

 - Address [comment](https://github.com/NREL/EnergyPlus/pull/8558#pullrequestreview-614855696) return mass flow, not volume flow.
 - Address [comment](https://github.com/NREL/EnergyPlus/pull/8558#pullrequestreview-614872281) use `CalcComponentSensibleLatentOutput` function.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Review by rraustad — APPROVED** [review]
This all looks reasonable.

File: src/EnergyPlus/Coils/CoilCoolingDX.cc [hunk 1]
```diff
@@ -574,9 +574,9 @@ CoilCoolingDXCurveFitSpeed &CoilCoolingDX::altModeNomSpeed()
 Real64 CoilCoolingDX::condMassFlowRate(bool const useAlternateMode)
 {
     if (useAlternateMode) {
-        return this->altModeNomSpeed().condenser_air_flow_rate;
+        return this->altModeNomSpeed().RatedCondAirMassFlowRate;
```
Comment (rraustad) [inline src/EnergyPlus/Coils/CoilCoolingDX.cc:577]: Why is this one PascalCase while most others are camelCase?

Comment (mjwitte) [reply src/EnergyPlus/Coils/CoilCoolingDX.cc:577]: Good question. Probably copy/paste from the old DX coil. We can discuss cleaning these up next week.

**Comment by mjwitte** [comment]
@rraustad Feel free to merge this, unless @Myoldmopar wants to review.

**Comment by Myoldmopar** [comment]
I’m operating in a weird state right now as the power keeps going on and off at my house so don’t wait on me 😄

**Pull request merged**